src/Table.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import allCountryScores from "./scores";
+import "./Table.css"
+
+function Table() {
```

Comment:
Is there a more descriptive name you could use for this component?

Reply:
Possibly the full name of the ticket I guess